### PR TITLE
Update for Pronto 0.6

### PR DESCRIPTION
**Pull request by zwaldowski**
- Automatically pass `--reporter json` to SwiftLint.
- Adapt for `Pronto::Runner` changes
- Fix parser test
- Add simple integration test

### CODE REVIEW

File: lib/pronto/swiftlint/wrapper.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ module Pronto
   module Swiftlint
     class Wrapper
       def lint
-        stdout, stderr, _ = Open3.capture3(swiftlint_executable)
+        stdout, stderr, _ = Open3.capture3("#{swiftlint_executable} lint --reporter json")
```
Comment (tapi) [inline lib/pronto/swiftlint/wrapper.rb:8]: Was gonna submit this myself.
Would love to see this merged for this change alone 😄

**Comment by tapi** [comment]
@ajanauskas Any chance of this PR being accepted and deployed?

**Comment by ajanauskas** [comment]
@tapi Sorry didn't notice pull request! Will check up tomorrow!

**Comment by ajanauskas** [comment]
@zwaldowski thanks for solving this and sorry for slow response time 😄

**Pull request merged**